django/db/migrations/state.py

```diff
@@ -135,6 +149,16 @@ def rename_model(self, app_label, old_name, new_name):
                 to_reload.add((model_state.app_label, model_state.name_lower))
         # Reload models related to old model before removing the old model.
         self.reload_models(to_reload, delay=True)
+        if 'all_relations' in self.__dict__:
+            old_name_key = (app_label, old_name.lower())
+            new_name_key = (app_label, new_name.lower())
+            if old_name_key in self.relations:
+                self.relations[new_name_key] = self.relations.pop(old_name_key)
+            for model_relations in self.relations.values():
+                if old_name_key in model_relations:
+                    model_relations[new_name_key] = model_relations.pop(old_name_key, '')
```

Comment:
We've checked that `old_name_key` is in the `model_relations` there is no need to pass a default value to `pop()`:
```suggestion
                    model_relations[new_name_key] = model_relations.pop(old_name_key)
```